pcdsdevices/jet.py

```diff
@@ -4,10 +4,50 @@
 from ophyd import Component as Cpt
 from ophyd import Device
 
-from .epics_motor import BeckhoffAxis
+from .component import UnrelatedComponent as UCpt
+from .epics_motor import IMS, BeckhoffAxis
 from .interface import BaseInterface
 
 
+class Injector(Device):
+    """
+    Positioner for liquid jet Injector.
+
+    Consists of 3 coarse control motors and 3 fine control motors.
+
+    Parameters
+    ----------
+    coarseX_prefix : str
+        Prefix for the coarse control motor in the X direction.
+
+    coarseY_prefix : str
+        Prefix for the coarse control motor in the Y direction.
+
+    coarseZ_prefix : str
+        Prefix for the coarse control motor in the Z direction.
+
+    fineX_prefix : str
+        Prefix for the fine control motor in the X direction.
+
+    fineY_prefix : str
+        Prefix for the fine control motor in the Y direction.
+
+    fineZ_prefix : str
+        Prefix for the fine control motor in the Z direction.
+    """
+
+    coarseX = UCpt(IMS)
+    coarseY = UCpt(IMS)
+    coarseZ = UCpt(IMS)
+    fineX = UCpt(IMS)
+    fineY = UCpt(IMS)
+    fineZ = UCpt(IMS)
```

Comment:
Note that not all jet manipulators are 6-axis.  Many are just 3 axes

Reply:
Hmm, ok I'll make another class